src/cli/test_command.zig

```diff
@@ -246,6 +252,19 @@ pub const CommandLineReporter = struct {
         this.summary.expectations += expectations;
         this.jest.tests.items(.status)[id] = TestRunner.Test.Status.todo;
     }
+
+    pub fn printSummary(this: *CommandLineReporter, summary: Summary, start_time: i128, total_tests: ?u32) void {
```

Comment:
Shouldn't this also print all the skipped/todo tests? also give the summary of 0 pass, 0 fail, etc.? That's what I would expect

Reply:
well the problem is the total_tests is not very total until all of the tests are run

Reply:
or did you mean the runned_tests variable?

Reply:
oh i got you